orthw

```diff
@@ -46,7 +46,7 @@ ort_config_package_configuration_dir="$configuration_home/package-configurations
 ort_config_package_curations_dir="$configuration_home/curations"
 ort_config_resolutions_file="$configuration_home/resolutions.yml"
 ort_config_rules_file="$configuration_home/evaluator-rules/src/main/resources/example.rules.kts"
-ort_config_custom_license_texts_dir="$configuration_home/custom-licenses"
+ort_config_custom_license_texts_dir="$configuration_home/custom-license-texts"
```

Comment:
As a side-question: Why does this script define all of these manually instead of just setting the top-level config directory via the `ORT_CONFIG_DIR` environment variable and then relying on ORT to resolve all these configuration files relative to that? That way you'd guard against such breakages in the future.

Reply:
This script is way older than `ort-config`.
So, these variables do have history.

Apart from that, I believe that it needs to be adjustable to different directory structure to support different user preferences.

Reply:
> This script is way older than `ort-config`.

Yes, the `ort-config` repository might have been created after the `orthw` script, but the `ORT_CONFIG_DIR` variable has been available for quite some time to point ORT at a non-default configuration directory location as a whole.

> Apart from that, I believe that it needs to be adjustable to different directory structure to support different user preferences.

Currently, if a user wants to adjust only a single configuration file location, (s)he must edit the `orthw` script to change the respective variable's value. That's not much different from using the `ORT_CONFIG_DIR` approach and then eventually requiring the user to add the respective command line overrides to the tool invocations. But I see how the latter might be a little bit more work for that (IMO rather exotic) use-case.